src/kOS.Safe/Encapsulation/Suffixes/VarArgsVoidSuffix.cs

```diff
@@ -0,0 +1,19 @@
+namespace kOS.Safe.Encapsulation.Suffixes
+{
+    public class VarArgsVoidSuffix<TParam> : SuffixBase
+    {
+        private readonly Del<TParam> del;
+
+        public delegate void Del<in TInnerParam>(params TInnerParam[] arguments);
+
+        public VarArgsVoidSuffix(Del<TParam> del, string description = "") : base(description)
+        {
+            this.del = del;
+        }
+
+        public override object Get()
+        {
+            return del;
+        }
+    }
+}
```

Comment:
I should point out that I inherited this from @tomekpiotrowski 's original branch but because of a redesign of the CALL suffix, it ended up never getting used yet.  We might want to just ditch this file entirely from the PR.